packages/flutter/test/material/tooltip_test.dart

```diff
@@ -198,7 +198,61 @@ void main() {
     );
     expect(tip.size.height, equals(24.0)); // 14.0 height + 5.0 padding * 2 (top, bottom)
     expect(tip.localToGlobal(tip.size.topLeft(Offset.zero)), equals(const Offset(10.0, 20.0)));
-  });
+  }, skip: kIsWeb && !isCanvasKit); // https://github.com/flutter/flutter/issues/99933
+
+  testWidgetsWithLeakTracking('Material3 - Does tooltip end up in the right place - top left', (WidgetTester tester) async {
+    final GlobalKey<TooltipState> tooltipKey = GlobalKey<TooltipState>();
+    late final OverlayEntry entry;
+    addTearDown(() => entry..remove()..dispose());
+
+    await tester.pumpWidget(
+      Directionality(
```

Comment:
Can we here replace this with MaterialApp? I know the theme data will fallback to M3 but I see another PR #139253 is trying to update the fallback value. I think using MaterialApp helps make sure this unit test is really for M3 no matter whether that PR is merged or not:)

Reply:
This makes sense. Thanks for suggesting this.